internal/backends/testing/lib.rs

```diff
@@ -51,6 +60,21 @@ impl i_slint_core::platform::Platform for TestingBackend {
             None
         }
     }
+
+    fn run_event_loop(&self) -> Result<(), PlatformError> {
+        loop {
+            let e = self.queue.0.lock().unwrap().pop_front();
+            match e {
+                Some(Event::Quit) => break Ok(()),
+                Some(Event::Event(e)) => e(),
+                None => std::thread::park(),
+            }
+        }
+    }
+
+    fn new_event_loop_proxy(&self) -> Option<Box<dyn i_slint_core::platform::EventLoopProxy>> {
+        Some(Box::new(self.queue.clone()))
```

Comment:
Since the preview backend didn't implement this, we could instentiate several of them (one per thread)
But now, if there are several thread, there will be one backend but because of all the thread_local variables we have all over slint, this will not work nice.  (in particular the fact that the backend is thread local)